src/main/kotlin/com/team4099/robot2023/subsystems/groundintake/GroundIntake.kt

```diff
@@ -176,10 +178,13 @@ class GroundIntake(private val io: GroundIntakeIO) : SubsystemBase() {
 
     Logger.getInstance().recordOutput("GroundIntake/currentState", currentState.name)
 
-    Logger.getInstance().recordOutput("GroundIntake/requestedState", requestedState.name)
+    Logger.getInstance()
+      .recordOutput("GroundIntake/requestedState", requestedState.javaClass.simpleName)
 
     Logger.getInstance()
-      .recordOutput("GroundIntake/isAtCommandedState", requestedState == currentState)
+      .recordOutput(
+        "GroundIntake/isAtCommandedState", currentState.equivalentToRequest(requestedState)
```

Comment:
This is fine here but again in other places (superstructure) this will not exist. The more important question here is per state (have we finished executing our motion profile, for the targeting position one, for example)